src/backend/mpi/write_buffer.hpp

```diff
@@ -265,6 +268,7 @@ class write_buffer
 		/**
 		 * @brief	Copy constructor
 		 * @param	other The write_buffer object to copy from
+		 * @note	with c++14 we could use std::shared_lock for lock_other
```

Comment:
We are requiring `c++17` !

Reply:
Removed the comment (including the one below). "Premature optimization".